projects/driver_controls/test/test_ads1015.c

```diff
@@ -106,25 +106,36 @@ void test_ads1015_read_invalid_input(void) {
                     ads1015_read_raw(&s_storage, NUM_ADS1015_CHANNELS, &reading));
 }
 
-// This test checks if the callbacks are called properly for enabled channels.
+/* This test checks if the callbacks are called properly for enabled channels.
```

Comment:
Why is this test gone?

Nit: IIRC we prefer `//` style comments to `/**/` It is relatively easy in most editors to just prepend `//` and it appears more explicit if you aren't using syntax highlighting.